packages/server/src/index.ts

```diff
@@ -147,3 +160,12 @@ export function verifyProperty<T>(
     },
   };
 }
+
+export function issueProperty<T>(
+  object: { [key: string]: string },
+  secret: SecretKey,
+  authType: AuthType,
+): Signature {
+  const properties = objectToProperty(object);
+  return authByAuthType(properties, secret, authType);
```

Comment:
Receives an object as an input and creates a property array and then authenticates this properties.
A function to issure properties from server.